<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWindow</class>
 <widget class="QWidget" name="FormWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Designer WebSocket Test</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>连接</string>
     </property>
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="editAddress">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ws://localhost:55441</string>
        </property>
        <property name="placeholderText">
         <string>比如：ws://localhost:55441</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonConnect">
        <property name="toolTip">
         <string>连接</string>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabApi">
      <attribute name="title">
       <string>API</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="0,0" rowminimumheight="1,1,4,1,1" columnminimumwidth="0,1">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="toolTip">
          <string>参数</string>
         </property>
         <property name="title">
          <string>Params</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="textEditParams">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="5">
        <widget class="QListWidget" name="listWidget">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Open File</string>
          </property>
          <property name="toolTip">
           <string>打开文件</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Close File</string>
          </property>
          <property name="toolTip">
           <string>关闭文件</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Window Info</string>
          </property>
          <property name="toolTip">
           <string>窗口信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ui Code</string>
          </property>
          <property name="toolTip">
           <string>窗口代码</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ScreenShot</string>
          </property>
          <property name="toolTip">
           <string>窗口截图</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Set StyleSheet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Set Properties</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Send Log</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBoxFormat">
            <property name="toolTip">
             <string>格式化json字符串</string>
            </property>
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxWrap">
            <property name="toolTip">
             <string>自动换行</string>
            </property>
            <property name="text">
             <string>Wrap</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>293</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonClear_2">
            <property name="toolTip">
             <string>清除</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextBrowser" name="textBrowserApi">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>376</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSend">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>发送</string>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEvent">
      <attribute name="title">
       <string>Event</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="textBrowserEvent"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClear">
         <property name="toolTip">
          <string>清除</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClear</sender>
   <signal>clicked()</signal>
   <receiver>textBrowserEvent</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClear_2</sender>
   <signal>clicked()</signal>
   <receiver>textBrowserApi</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
